core/src/test/java/org/infinispan/configuration/ConfigurationUnitTest.java

```diff
@@ -81,7 +81,7 @@ public void testDistSyncAutoCommit() {
    }
 
    @Test
-   public void testDummyTMGetCache() throws Exception {
```

Comment:
-1 for removing `throws` clauses in tests, it's too much noise adding and removing exceptions all the time.

You can edit the inspection settings in IntelliJ to "Ignore exceptions thrown by entry point methods" instead.

Reply:
It's already done. And I could probably tolerate those in tests but not production code. -1 for crying over spilled milk

Reply:
Since the changes are just noise, what's some more noise on top of it in the form of PR comments? :P

Reply:
More please!